pkgs/top-level/all-packages.nix

```diff
@@ -15023,12 +15023,6 @@ in
 
   pam_usb = callPackage ../os-specific/linux/pam_usb { };
 
-  paxctl = callPackage ../os-specific/linux/paxctl { };
-
-  paxtest = callPackage ../os-specific/linux/paxtest { };
```

Comment:
I think that leaving at least paxtest and maybe even paxctl makes sense; the former can be used to assess whether vanilla mitigations are able top catch stuff covered by PaX, which I find useful (given the ongoing effort to backport/lift features from grsec/pax).